scene/gui/line_edit.cpp

```diff
@@ -769,18 +783,6 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 	}
 
 	if (is_shortcut_keys_enabled()) {
-		if (k->is_action("ui_copy", true)) {
-			copy_text();
-			accept_event();
-			return;
-		}
-
-		if (k->is_action("ui_text_select_all", true)) {
-			select();
-			accept_event();
-			return;
-		}
-
 		// Cut / Paste
 		if (k->is_action("ui_cut", true)) {
```

Comment:
Cut should copy when not editable, so it matches with TextEdit behavior.
The `cut_text()` method can modified for this.

Reply:
IMO the TextEdit's behavior is wrong.

Reply:
The text in your example gets copied on Windows / Firefox using `Ctrl+c`.

Reply:
Yes, that's Copy. Ctrl+X does nothing.

Reply:
Sorry, I meant using `Ctrl+X`

Reply:
Indeed, but it does not on Chrome 🤔

Reply:
It also does not on MS Edge

Reply:
> The text in your example gets copied on Windows / Firefox using `Ctrl+x`.

Interesting, on Linux / Firefox `Ctrl+X` does not seem to copy.